frontend/src/Form.js

```diff
@@ -0,0 +1,51 @@
+import React, {useState} from 'react';

```

Comment:
This file is a copy of CreateAccount as well?

Reply:
@RandallCaler